package.json

```diff
@@ -42,16 +42,16 @@
     "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
+    "danger": "^3.4.5",
     "enzyme": "^3.2.0",
     "enzyme-adapter-react-16": "^1.1.0",
     "eslint": "^4.12.1",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.5.1",
+    "execa": "^0.10.0",
```

Comment:
Because `child_process.exec` doesn't put the local `bin` in `node_modules` in the path by default, and `execa` handles the windows compat problems